tidy3d/components/spice/analysis/dc.py

```diff
@@ -0,0 +1,62 @@
+"""
+This class defines standard SPICE analysis types (electrical simulations configurations).
+"""
+
+from typing import Optional
+
+import pydantic.v1 as pd
+
+from tidy3d.components.base import Tidy3dBaseModel
+from tidy3d.components.spice.sources.dc import MultiStaticTransferSourceDC, StaticTransferSourceDC
+
+
+class OperatingPointDC(Tidy3dBaseModel):
+    """
+    Equivalent to Section 11.1.2 in the ngspice manual.
+    """
+
+
+class TransferFunctionDC(Tidy3dBaseModel):
+    """This class sets parameters used in DC simulations.
+
+    Ultimately, equivalent to Section 11.3.2 in the ngspice manual.
```

Comment:
I'm not sure about this name. It sort of implies we get the response of a device/circuit for a range of input values (e.g., voltage) which may not be the case? We could be interested in a single point operating point?